az-digital/az_quickstart
# Closes #1651: Add grayscale background color options

## Description
Adds brand-approved neutral gray colors to text with background, split screen, and card paragraph types.

## Related issues
Closes #1651 

## How to test

1. Add card paragraph type to a page
2. Select card background color of Gray 100, Gray 200, or Gray 300
3. Add split screen paragraph type to a page
4. Select background color of Gray 100, Gray 200, or Gray 300
5. Add text with background paragraph type to a page
6. Select background color of Gray 100, Gray 200, or Gray 300
7. Test these paragraph types with other settings to ensure no unintentional changes

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
